systemtests/src/test/java/org/bf2/admin/kafka/systemtest/oauth/ConsumerGroupsOAuthTestIT.java

```diff
@@ -38,14 +42,26 @@
     @Inject
     Config config;
 
-    TokenUtils tokenUtils;
+    static TokenUtils tokenUtils;
+    static String initialToken;
+    static long initialTokenExpiresAt;
+
     TopicUtils topicUtils;
     ConsumerUtils groupUtils;
     String batchId;
 
+    @BeforeAll
+    static void initialize() {
+        tokenUtils = new TokenUtils(ConfigProvider.getConfig());
+        JsonObject token = tokenUtils.getTokenObject(UserType.OWNER.getUsername());
+
+        // Fetch one token before the tests begin to minimize wait time in expired token test
+        initialToken = token.getString("access_token");
+        initialTokenExpiresAt = System.currentTimeMillis() + (token.getInt("expires_in") * 1000);
```

Comment:
Roughly what is the token duration in the test? 

Is there any way to shorten the duration for the test or inject a [Clock](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/Clock.html) so we can control time inside the tests?

Reply:
> Roughly what is the token duration in the test?

The token lifespan is 30s (down from 120s in the `main` branch). I've looked at this (admittedly not exhaustively) and I don't believe it can be shorted for individual tokens. I've considered ordering the tests to make sure this one runs last (i.e. the wait time is minimized), but that's still on the "to do" list.
https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/e1a38b2566ffec455627da43a2cdde1286ec73f2/systemtests/src/main/resources/keycloak-import/realms/authz-realm.json#L3